chapters/6-analysis.tex

```diff
@@ -217,59 +214,66 @@ \subsection{Integrating different data sources}
 % -------------------------------------------------------------------------
 
 % Data integration
-Note, however, that not all merges of data with different units of observation 
-are as conceptually straightforward.
-More complex cases may arise if you are looking to overlay road location data with household data,
-using a spatial match;
-to combine school administrative data, such as attendance records and test scores,
-with household demographic characteristics from a survey;
-or link a dataset of infrastructure access points, such as water pumps or schools,
-and a dataset of household locations.
-In these cases, a key part of the research contribution is figuring out what
-a useful way to combine the datasets is.
+Some merges of data with different units of observation
+are more conceptually complex.
+Some examples include: overlaying road location data with household data,
+using a spatial match; combining school administrative data, such as attendance records and test scores,
+with student demographic characteristics from a survey;
+or linking a dataset of infrastructure access points, such as water pumps or schools,
+with a dataset of household locations.
+In these cases, a key part of the research contribution is figuring out
+a useful way to combine the datasets.
 Since the conceptual constructs that link observations from the two data sources
-are so important and can take many possible forms,
+are important and can take many possible forms,
 it is especially important for the data integration to not be treated mechanically,
-and to be extensively documented separately from other data construction tasks.
+and to be extensively documented, separately from other data construction tasks.
```

Comment:
Discussed in chapter 3 in relation to data linkage tables.

Reply:
Done in 06f9055